Modules/Core/Common/include/itkArray.h

```diff
@@ -100,6 +100,7 @@ class ITK_TEMPLATE_EXPORT Array : public vnl_vector<TValue>
 #endif
 
   /** Constructor to initialize an array from another of any data type */
+  /** @ITKStartGrouping */
```

Comment:
We don't want grouping specifications for single declarations such as this.

Reply:
I agree, see also my previous and next comment.